app/components/user/graphql/GraphQLExplorerConsole.js

```diff
@@ -282,6 +291,12 @@ class GraphQLExplorerConsole extends React.PureComponent<Props, State> {
     this.executeCurrentQuery();
   };
 
+  handlePrettifyClick = () => {
+    prettifyQuery(this.state.query, (query) => {
+      this.setState({ query: query });
+    });
+  }
```

Comment:
I wonder if we’d want to do the lazy-load-on-hover trick here? I did it in ImageUploadField:

https://github.com/buildkite/frontend/blob/bb8aaed01cb7b16aa2aaff4d6ffec8280daf9891/app/components/shared/ImageUploadField.js#L205-L219